pkg/kubelet/container/sync_result.go

```diff
@@ -68,6 +70,8 @@ const (
 	ConfigPodSandbox SyncAction = "ConfigPodSandbox"
 	// KillPodSandbox action
 	KillPodSandbox SyncAction = "KillPodSandbox"
+	// ResizePod action
+	ResizePod SyncAction = "ResizePod"
```

Comment:
nit: Maybe add "InPlace" to be a little more precise? We don't call the doPodResizeAction method when the containers use a `RestartContainer` resize policy.

```suggestion
	ResizePodInPlace SyncAction = "ResizePodInPlace"
```

Reply:
done!